app/src/main/java/wannabit/io/cosmostaion/dialog/SwapCoinListDialog.java

```diff
@@ -191,23 +192,44 @@ public void onClick(View view) {
         }
 
         private void onBindSelectedChainListItemViewHolder(SwapChainHolder holder, int position) {
-            BaseChain baseChain = mWatchAddressChainList.get(position);
+            BaseChain baseChain = WDp.getChainsFromAddress(mWatchAddress).get(position);
             ChainConfig chainConfig = ChainFactory.getChain(baseChain);
             WDp.setDpSymbolImg(getSActivity().getBaseDao(), chainConfig, chainConfig.mainDenom(), holder.coinImg);
             WDp.setDpSymbol(getSActivity(), getSActivity().getBaseDao(), chainConfig, chainConfig.mainDenom(), holder.coinName);
 
-            bindChainSelect(holder, position, baseChain);
+            ArrayList<Account> watchAddressAccounts = getSActivity().getBaseDao().onSelectAccountsByChain(baseChain);
+            for (Account account : watchAddressAccounts) {
```

Comment:
for (Account account : watchAddressAccounts) {
                if ((chainConfig.baseChain().equals(BaseChain.OKEX_MAIN) && account.address.equalsIgnoreCase(mWatchAddress))
                        || account.address.equalsIgnoreCase(WKey.convertAddressEthToTender(chainConfig.baseChain(), mWatchAddress))) {
                    holder.rootLayer.setClickable(false);
                    holder.rootLayer.setBackground(ContextCompat.getDrawable(getSActivity(), R.drawable.box_round_gray));
                    holder.rootDimLayer.setVisibility(View.VISIBLE);
                    holder.rootDimLayer.setAlpha(0.5f);
                } else {
                    bindChainSelect(holder, position, baseChain);
                }
            }